src/mpi4py/MPI.src/asdlpack.pxi

```diff
@@ -189,17 +205,22 @@ cdef int Py_CheckDLPackBuffer(object obj) noexcept:
     except: return 0  # ~> uncovered  # noqa
 
 cdef int Py_GetDLPackBuffer(object obj, Py_buffer *view, int flags) except -1:
+    cdef int version = 1
+    cdef const char *capsulename = b"dltensor_versioned"
```

Comment:
@leofang Please confirm the capsule names corresponding to v1.0 need a "_versioned" prefix.